### PR TITLE
Added a <p> in the body

### PR DESCRIPTION
**Pull request by BryanGC96**

### CODE REVIEW

**Review by lage356 — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (montermarco) [inline index.html:6]: Please change the Document Title

**Review by montermarco — APPROVED** [review]
LGTM

**Comment by montermarco** [comment]
LGTM!

**Pull request merged**